packages/shared/src/domain_model/WriteIn.ts

```diff
@@ -0,0 +1,12 @@
+import { Uid } from "./Uid";
```

Comment:
Note that @recursivesquircle is working on shorter IDs at the moment. 
https://github.com/Equal-Vote/star-server/issues/733

Depending on when this is merged you may need to rebase so that the race id uses the new format

Reply:
Good to know and I'll keep an eye out, but this is just the Uid type which is just a string which I doubt will change.